core/src/com/unciv/logic/city/CityConstructions.kt

```diff
@@ -419,6 +421,10 @@ class CityConstructions : IsPartOfGameInfoSerialization {
                 else
                     civ.addNotification("[${construction.name}] has been built in a faraway land", buildingIcon)
             }
+
+            if (cityInfo.civInfo.isPlayerCivilization() && (cityInfo.civInfo.gameInfo.currentPlayerCiv == cityInfo.civInfo)) {
```

Comment:
Constructions usually complete at the *end* of your turn, but the wonder popup appears at the start of your next turn.
How about we move the music trigger to the wonder popup instead?

Reply:
Oh good point, plus we won't need the if statement.